CMakeLists.txt

```diff
@@ -118,6 +118,10 @@ IF (DEPRECATE_HARD)
 	ADD_DEFINITIONS(-DGIT_DEPRECATE_HARD)
 ENDIF()
 
+IF (NOT BUILD_SHARED_LIBS)
+	ADD_DEFINITIONS(-DGIT_STATIC)
+ENDIF()
```

Comment:
I think I'd prefer if this was present in "src/features.h.in" to make the definition available in "global.h".